app/routes/capital-projects.$managingCode.$capitalProjectId.tsx

```diff
@@ -1,3 +1,43 @@
+import { LoaderFunctionArgs, json } from "@remix-run/node";
+import { useLoaderData, useNavigate, useSearchParams } from "@remix-run/react";
+import {
+  findCapitalProjectByManagingCodeCapitalProjectId,
+  findAgencies,
+} from "../gen";
+import { CapitalProjectDetailPanel } from "../components/CapitalProjectDetailPanel";
+
+export async function loader({ params }: LoaderFunctionArgs) {
+  const { managingCode, capitalProjectId } = params;
+  const agenciesResponse = await findAgencies({
+    baseURL: `${import.meta.env.VITE_ZONING_API_URL}/api`,
```

Comment:
There's gotta be a way to configure the baseUrl so that we don't need to state it on each call.

Reply:
Issue #46